doc/common/Cadl-Project-Scripts.md

```diff
@@ -0,0 +1,127 @@
+# Integrating with cadl sync and generate scripts
+
+There are 2 common scripts provided for each language to be able to generate from within the language
+repo and use the remote cadl definition in the spec repo.
+
+## Prerequisites
```

Comment:
I'd expect the prerequisites to be a one time setup and will most likely not be needed by any library developers. While this was super useful for me to get things setup as a first time user, it may not be needed for others who are just interested in getting their library generated. We can consider having this prerequisites section pull out into a separate .md file for setting up the language repo for the first time.

Reply:
Instead of moving it do you think renaming it to `One time language repo setup` would make it more clear?

Reply:
Yeah, changing the section title works too. Helps the lib devs jump past the section that they don't have to ever read or take action.